packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -203,7 +203,10 @@ - (void)_createUpdateWithManifest:(NSMutableDictionary *)mutableManifest
                      successBlock:(EXUpdatesFileDownloaderManifestSuccessBlock)successBlock
                        errorBlock:(EXUpdatesFileDownloaderErrorBlock)errorBlock
 {
-  mutableManifest[@"isVerified"] = @(isVerified);
+  if ( _config.expectsSignedManifest && !isVerified ){
```

Comment:
```suggestion
  if (_config.expectsSignedManifest) {
```
we'll want to add this key in both cases (verified and unverified) for Expo Go.

Reply:
OK was a little confused and reread the android code.

In the android code `isVerified` is always set if it is `true` and only set to `false` if it is also expecting a signed manifest.

So if we want that we should actually switch this to:
`if ( _config.expectsSignedManifest || isVerified ){`

(the iOS code differs a bit in that we are editing  a setter function instead of inline code)

Reply:
(link to android PR: https://github.com/expo/expo/pull/12027/files#diff-2e2ccf9c4e723912462ae8b4f17dd87433be7cb5ede026d320edabad1504b797R169)